sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/transaction/PersistentTransactionManager.kt

```diff
@@ -145,8 +145,10 @@ class PersistentTransactionManager(
         tx
     }
 
+    // TODO(str4d): This uses operator overloading to distinguish between it and createToAddress, which breaks when
+    //  spendingKey is removed. Figure out where these methods need to be renamed, and do so.
     override suspend fun encode(
```

Comment:
I'm still figuring out how these APIs and their overloading work. We can't merge this PR until we fix this overloading, due to @nuttycom's desire to move all of these APIs to use the (intentionally unstable) binary USK encoding (which will make these two APIs have identical signatures).

Reply:
You should just use the USK because these APIs are really internal I don't think that maintaining the String based API will do any good

Reply:
PersistentTransactionManager is removed with Roomoval, so I wouldn't worry too much about it.  It will be removed from the public API entirely.